.github/workflows/go.yml

```diff
@@ -31,12 +57,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Run go test
-        run: go test -coverprofile=coverage.txt -covermode=atomic ./...
-
-      - name: Codecov
-        env:
-          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
-        run: bash <(curl -s https://codecov.io/bash)
+        run: go test -race ./...
```

Comment:
why `race` here now?

Reply:
It should've always been this way. `-race` is good.

Reply:
start of racing is a separate change to me

Reply:
Fixed.